feature-set/src/lib.rs

```diff
@@ -98,6 +99,65 @@ impl FeatureSet {
         self.activated_slot(&reduce_stake_warmup_cooldown::id())
             .map(|slot| epoch_schedule.get_epoch(slot))
     }
+
+    pub fn runtime_features(&self) -> RuntimeFeatures {
+        RuntimeFeatures {
+            lift_cpi_caller_restriction: self.activated_slot(&lift_cpi_caller_restriction::id()),
```

Comment:
This seems like a lot of repeated code. Do you think we can abstract this initialization in a macro? The struct members are named the same as the features.

Reply:
It'll remove the possibility of naming the feature ID with a different name than what we call in SVM repo. It's easier to call it the same since the features already exist. But, when SVM is a separate repo, it might be extra work to enforce it. 

We can probably add a macro in a separate PR, so if we don't like the rigidity in the future we can just revert that PR.